.golangci.yml

```diff
@@ -0,0 +1,59 @@
+run:
+  timeout: 5m
+  modules-download-mode: readonly
+
+linters-settings:
+  goconst:
+    min-len: 2
+    min-occurrences: 2
+  gofmt:
+    simplify: true
+  goimports:
+    local-prefixes: github.com/mattermost/mattermost-plugin-aws-SNS
+  golint:
+    min-confidence: 0
+  govet:
+    check-shadowing: true
+    enable-all: true
+  misspell:
+    locale: US
+
+linters:
+  disable-all: true
+  enable:
+    - bodyclose
+    - deadcode
+    - errcheck
+    - goconst
+    - gocritic
+    - gofmt
+    - goimports
+    - golint
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - misspell
+    - nakedret
+    - staticcheck
+    - structcheck
+    - stylecheck
+    - typecheck
+    - unconvert
+    - unused
+    - varcheck
+    - whitespace
+
+issues:
+  exclude-rules:
+    - path: server/manifest.go
+      linters:
+        - deadcode
+        - varcheck
+    - path: server/fetcher.go
```

Comment:
You can just remove `fetchOrgMember` instead. It's not used.

Reply:
Okay I'll fix these, thanks

Reply:
Ended up making the github rate limit check a separate function to be shared. I see some other areas with some duplication as well and will look at trying to clean some of that up.
From an organization standpoint where should I put `githubErrorHandle()`?

Reply:
A new file `github.go` would be a good place.